### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by ghost** [LilithWittmann/pepper-tablet-hack#1]
Hi,
your work is really interesting, unfortunately when I run the project on my Pepper, the tablet restart saw 28 installations and  1 min after I get the background by default. I cannot modify the launcher or install any application on the tablet.
(my NaoQi version is 2.5.7.1)

**Comment by LilithWittmann** [comment]
Hey, sorry but I don't have access to the current version of Pepper. Maybe SBR has changed some of their "security" features.

**Comment by soerent** [comment]
I get a Java Exeception when i execute the Kingo Root App - something you have seen before

**Comment by LilithWittmann** [comment]
Yeah I have! Could you post a picture of the errormessage? (Sorry I don't have access to a pepper robot with one of the latest NaoQI versions, so it's not possible to test it, but maybe I could find out whats your issue with the exact errormessage - or via screen sharing/…)
